Tools/TestWebKitAPI/Tests/WebKitCocoa/ScreenTime.mm

```diff
@@ -507,7 +522,12 @@ - (void)userContentController:(WKUserContentController *)userContentController d
     [[window contentView] addSubview:webView.get()];
     [window makeKeyAndOrderFront:nil];
 
-    RetainPtr request = [NSURLRequest requestWithURL:[NSBundle.test_resourcesBundle URLForResource:@"PictureInPictureDelegate" withExtension:@"html"]];
+    // First, load a URL with a HTTP protocol to install the controller.
```

Comment:
Rather than a test workaround, it might be slightly nicer to use `HTTPServer` to test your content with an HTTP URL. Note that `loadSimulatedRequest:response:responseData:` won't work in this case since there is a subresource (the video).

Will share a diff.